jsonschema/_validators.py

```diff
@@ -70,13 +73,31 @@ def items(validator, items, instance, schema):
     if not validator.is_type(instance, "array"):
         return
 
-    if validator.is_type(items, "array"):
-        for (index, item), subschema in zip(enumerate(instance), items):
+    if validator.is_type(items, "boolean") and 'prefixItems' in schema:
+        if not items:
+            if len(instance) > len(schema['prefixItems']):
+                yield ValidationError(
+                    "%r has more items than defined in prefixItems" % instance
+                )
+            else:
+                for error in validator.descend(
+                        instance,
+                        {'prefixItems': schema['prefixItems']},
+                        path='items__prefixItems',
```

Comment:
This path doesn't look right -- it should be an indexable thing in the instance -- i.e. `instance[path]` should exist.

(I honestly don't know yet what prefixItems does though, so I'll have to check before making a specific recommendation, but if I can guess at what it does, the path should be a list index, probably the first one that's invalid)

Reply:
After refactoring I noticed that the `descend` is not reuired here, as this is already handled in the `prefixItems` function.